x-pack/plugins/endpoint/public/applications/endpoint/store/managing/middleware.ts

```diff
@@ -32,5 +41,15 @@ export const managementMiddlewareFactory: MiddlewareFactory<ManagementListState>
         payload: response,
       });
     }
+    if (action.type === 'userChangedUrl' && hasSelectedHost(state) !== undefined) {
```

Comment:
I think `hasSelectedHost()` returns a `boolean`. Why is the conditional checking for `undefined`?